workflow/snakemake_rules/nextclade.smk

```diff
@@ -179,18 +180,19 @@ rule run_nextclade:
     params:
         genes=GENES_SPACE_DELIMITED,
         translation_arg=lambda w: (
-            f"--output-translations=data/{database}/nextclade{w.reference}.translation_{{gene}}.upd.fasta"
+            # Nextclade takes a filename template in which it replaces {gene}
+            # itself, so we want to pass thru {gene} literally to it and make
+            # sure it isn't interpretted by the shell as a glob.  We shellquote
+            # here instead of with :q below because we don't want to pass an
+            # empty string argument when this param is empty.
+            shellquote(f"--output-translations=data/{database}/nextclade{w.reference}.translation_{{gene}}.upd.fasta")
```

Comment:
A minor imprecision in my comment: `{...}` is a shell expansion (specifically brace expansion) not a shell glob.

Shell quoting may not have been an issue in practice, as the brace expansion in Bash only applies when there's a sequence, e.g. `{a,b}` and not a single item `{a}`.

Reply:
Slightly off topic as Nextclade related but worth having the context here

Pinging @ivan-aksamentov as he devised the template format (I think) :)

We haven't had issues with this yet (I think), but given we heavily use snakemake with nextclade, we may want to template things slightly differently in the future? @tsibley do you have a suggestion for how else we could specify the template string to be less of a pain with snakemake?